src/compressAlignment.hpp

```diff
@@ -77,26 +77,32 @@ that can be represented                    3       7      511   -2^19   -2^31
 
 // Standard library.
 #include "string.hpp"
-#include "span.hpp"
 
 
 namespace shasta {
     void compress(const Alignment&, string&);
-    void decompress(span<const char>, Alignment);
+    void decompress(const string&, Alignment&);
+
+    void testAlignmentCompression();
 
     namespace compressAlignment {
         class Format0;
         class Format1;
         class Format2;
         class Format3;
         class Format4;
+
+        uint8_t extractFormatIdentifier(const char);
     }
 }
 
 
 
 class shasta::compressAlignment::Format0 {
 public:
+    static const uint8_t ID = 0b0;
```

Comment:
I don't use uppercase for static constants (`ID, ID_MASK`). I use the same style as for other constants (`id, idMask`).

Reply:
Cool.